config_manager.go

```diff
@@ -35,6 +37,16 @@ type ConfigFile struct {
 	ExcludePatterns *RegexCollection `yaml:"exclude_patterns"`
 }
 
+func (cf ConfigFile) printValidationWarnings() {
+	if len(cf.Files) == 0 {
+		log.Warningf("Validation: configuration file contains no files to watch")
+	}
+	// Warn, that port is not set, only if host is specified
+	if cf.Destination.Host != "" && cf.Destination.Port == 0 {
+		log.Warningf("Validation: destination port is not specified in configuration file")
```

Comment:
Also, I am not sure the port could ever end up being zero given the behaviour in [DestPort()](https://github.com/featalion/remote_syslog2/blob/fix-issue-43/config_manager.go#L265), which seems to default to `514`. 

Is the above check necessary?